src/world/area_kkj/kkj_23/B03190.c

```diff
@@ -1,5 +1,23 @@
 #include "kkj_23.h"
+#include "effects.h"
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_23/B03190", func_80240910_B03190);
+void func_800E63A4(s32);
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_23/B03190", func_802409AC_B0322C);
+ApiStatus func_80240910_B03190(Evt* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    EffectInstance* effect = (EffectInstance*) evt_get_variable(script, *args++);
+    s32 var1 = evt_get_variable(script, *args++);
+    s32 var2 = evt_get_variable(script, *args++);
+    s32 var3 = evt_get_variable(script, *args++);
+
+    effect->data.any[12] = var1;
+    effect->data.any[13] = var2;
+    effect->data.any[14] = var3;
```

Comment:
Still can't tell for sure which `FXData` we're using here. This file does not have much more information.

Reply:
LightningFXData